packages/salesforcedx-vscode-core/src/telemetry/telemetry.ts

```diff
@@ -190,8 +190,8 @@ export class TelemetryService {
     }
   }
 
-  private getEndHRTime(hrstart: [number, number]): string {
+  public getEndHRTime(hrstart: [number, number]): string {
     const hrend = process.hrtime(hrstart);
-    return util.format('%d%d', hrend[0], hrend[1] / 1000000);
+    return util.format('%d.%d', hrend[0], Math.round(hrend[1] / 1000000));
```

Comment:
I think for this we were concatenating the milliseconds to the seconds. 5s and 430.123ms would look something like 5430.123 instead of 5.430

Reply:
Let's leave this change in format out of this PR since it will also change the way we measure all current perf info and dashboards.

Reply:
Doh, my bad I didn't notice the dashboard were in ms.